runtimes/eden/src/xcm_config.rs

```diff
@@ -151,3 +162,76 @@ impl cumulus_pallet_xcm::Config for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
 }
+pub struct AccountIdToMultiLocation;
+impl Convert<AccountId, MultiLocation> for AccountIdToMultiLocation {
+	fn convert(account: AccountId) -> MultiLocation {
+		X1(AccountId32 {
+			network: NetworkId::Any,
+			id: account.into(),
+		})
+		.into()
+	}
+}
+parameter_types! {
+	pub const BaseXcmWeight: XcmWeight = 100_000_000; // TODO: recheck this
+	pub const MaxAssetsForTransfer: usize = 2;
+}
+parameter_types! {
+	pub SelfLocation: MultiLocation = MultiLocation::here();
+}
+parameter_type_with_key! {
+	pub ParachainMinFee: |_location: MultiLocation| -> Option<u128> {
+		Some(u128::MAX)
```

Comment:
Maximising the minimum fee doesn't seem right to me..

Reply:
it can also be set to None: 

> If Parachain don't want have this case, can simply return None. A default implementation is provided by DisabledParachainFee in xcm-support.

Reply:
I committed the None case